health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/RenderConditionalField.js

```
@@ -0,0 +1,220 @@
+      const mdmsOptions = [
+        {
+          moduleName: "common-masters",
+          masterName: "GenderType",
+          schemaCode: "common-masters.GenderType",
+        },
+        {
+          moduleName: "HCM",
+          masterName: "HOUSE_STRUCTURE_TYPES",
+          schemaCode: "HCM.HOUSE_STRUCTURE_TYPES",
+        },
+      ];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)